tests/openadapt/test_events.py

```diff
@@ -202,58 +217,52 @@ def make_doubleclick_event(
 
 def test_merge_consecutive_mouse_click_events():
     if OVERRIDE_DOUBLE_CLICK_INTERVAL_SECONDS:
-        override_double_click_interval_seconds(
-            OVERRIDE_DOUBLE_CLICK_INTERVAL_SECONDS
-        )
+        override_double_click_interval_seconds(OVERRIDE_DOUBLE_CLICK_INTERVAL_SECONDS)
     double_click_interval_seconds = get_double_click_interval_seconds()
     dt_short = double_click_interval_seconds / 10
     dt_long = double_click_interval_seconds * 10
     raw_events = [
         *make_click_events(dt_long, button_name="right"),
-
```

Comment:
I find these blank spaces to make the test more readable. @Mustaballer can you please add directives here to allow these lines?